stacks/upload-db-stack.js

```diff
@@ -28,16 +30,21 @@ export function UploadDbStack({ stack, app }) {
   // Upload API private key
   const privateKey = new Config.Secret(stack, 'PRIVATE_KEY')
 
-  // Content claims private key
-  // TODO: we should look into creating a trust layer for content claims
-  const contentClaimsPrivateKey = new Config.Secret(stack, 'CONTENT_CLAIMS_PRIVATE_KEY')
+  // Not strictly a secret, but it makes the env vars exceed the 4kb limit...
+  const indexingServiceProof = new Config.Secret(stack, 'INDEXING_SERVICE_PROOF')
 
   /**
    * The allocation table tracks allocated multihashes per space.
    * Used by the blob/* service capabilities.
    */
   const allocationTable = new Table(stack, 'allocation', allocationTableProps)
 
+  /**
+   * The blob registry table contains information about blob registrations
+   * per space.
+   */
+  const blobRegistryTable = new Table(stack, 'blob-registry', blobRegistryTableProps)
```

Comment:
This table will eventually replace the `allocation` table after migration.

Mostly a rename, but it is also written to at different points in the upload flow. The `allocation` table was written to after `(web3.storage/)blob/allocate` was invoked, _before_ the file was received. The `blob-registry` is written to after a successful `blob/accept` invocation.